third_party/cyw30739_sdk/merge_hex.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
```

Comment:
license blurb

also comment on what it does.

Reply:
done.